config/locales/cy.yml

```diff
@@ -188,6 +188,43 @@ toplevel:
       description: <p>Diolch i chwaraewyr y <a href="%s">Loteri Genedlaethol</a> sy’n codi arian ar gyfer achosion da, rydym yn dosbarthu mwy na <a href="%s">£500 miliwn</a> i gymunedau ledled y Deyrnas Unedig bob blwyddyn</p>
     under10k: Ymgeisio am grant o dan £10,000
     over10k: Ymgeisio am grant dros £10,000
+    covid19Statement:
+      title: Ein hariannu yn ystod COVID-19
+      body: >
+        <p><strong>Beth rydych angen ei wybod</strong><br/></p>
+        <p>
+            Yn gyntaf, rydym eisiau i chi wybod y byddwn yn parhau
+            i anrhydeddu ein hymrwymiadau cyfredol ac yn parhau
+            i fod yn hyblyg gyda'r grantiau presennol.
+            Fodd bynnag, COVID-19 fydd ein blaenoriaeth.
+        </p>
+        <p><strong>Beth mae hyn yn ei olygu</strong><br/></p>
+        <p>
+            Bydd yr holl benderfyniadau ariannu a wnawn am y
+            chwe mis nesaf yn blaenoriaethu cael arian grant i
+            grwpiau sydd yn y sefyllfa orau i gefnogi eu cymunedau
+            ar yr adeg hanfodol hon gan ganolbwyntio i ddechrau ar
+            ddeiliaid grantiau ac ymgeiswyr presennol.
+        </p>
+        <p><strong>Byddwn yn blaenoriaethu:</strong><br/></p>
+        <ul>
+            <li>Mudiadau sy'n cefnogi pobl sydd â risg uchel o COVID-19</li>
+            <li>Mudiadau sy'n cefnogi cymunedau sydd fwyaf tebygol o wynebu galw a heriau cynyddol o ganlyniad uniongyrchol o COVID-19</li>
+            <li>Mudiadau sydd â photensial uchel i gefnogi cymunedau ag effaith uniongyrchol ac anuniongyrchol COVID-19</li>
+        </ul>
+        <p>
+          Os hoffech chi gyflwyno cais nad yw'n gysylltiedig â COVID-19,
+          mae croeso i chi wneud hynny o hyd. Fodd bynnag, byddwch yn
+          ymwybodol ei bod yn debygol y bydd mwy o gystadleuaeth am ein harian grant,
+          ac oediad yn ein gallu i ymateb.
+
+          Os ydych eisoes wedi cyflwyno cais nad yw'n gysylltiedig â COVID,
+          byddwn mewn cysylltiad i drafod y cais penodol hwn gyda chi.
+
+          Sylwch y byddwn yn diweddaru canllawiau ar ein tudalennau
+          rhaglen benodol ar ein gwefan ddydd Llun.
+        </p>
+      linkText: Darllenwch y diweddariadau diwethaf ar ein tudalen COVID-19
```

Comment:
Missing translation so used "Read the latest updates on our COVID-19 page" rather than the shorter "Read our latest COVID-19 updates". Using as an appropriate placeholder.